third-party-synthetic/active-gate-extensions/extension-third-party-ping/src/ping_extension.py

```diff
@@ -18,6 +18,8 @@ def initialize(self, **kwargs):
             self.config.get("api_url"), self.config.get("api_token"), log=log, proxies=self.build_proxy_url()
         )
         self.executions = 0
+        self.failure_detected = 0
```

Comment:
This should be `self.failures_detected` since it represents the total number of failures

Reply:
ok